target/linux/generic/hack-5.4/662-remove_pfifo_fast.patch

```diff
@@ -1,224 +0,0 @@
-From b531d492d5ef1cf9dba0f4888eb5fd8624a6d762 Mon Sep 17 00:00:00 2001
-From: Felix Fietkau <nbd@nbd.name>
-Date: Fri, 7 Jul 2017 17:23:42 +0200
-Subject: net: sched: switch default qdisc from pfifo_fast to fq_codel and remove pfifo_fast
-
-Signed-off-by: Felix Fietkau <nbd@nbd.name>
----
- net/sched/sch_generic.c | 140 ------------------------------------------------
- 1 file changed, 140 deletions(-)
-
---- a/net/sched/sch_generic.c
-+++ b/net/sched/sch_generic.c
```

Comment:
Why did you remove this patch?
The upstream changes are not looking like they integrated this functionality:
https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/diff/net/sched/sch_generic.c?id=v5.4.124&id2=v5.4.123

Reply:
Ah, I see this now.  My mistake was believing the quilt output:
```
% quilt push
Applying patch generic-hack/662-remove_pfifo_fast.patch
patching file net/sched/sch_generic.c
Hunk #1 FAILED at 595.
1 out of 1 hunk FAILED -- rejects in file net/sched/sch_generic.c
Patch generic-hack/662-remove_pfifo_fast.patch can be reverse-applied
```
I see now that it needed a refresh.  I will re-build/verify and fpush when good.